native/SampleApps/MobileSyncExplorer/MobileSyncExplorer/Classes/ContactListViewController.m

```diff
@@ -170,13 +170,17 @@ - (void)loadView {
 }
 
 - (void)viewWillLayoutSubviews {
-    CGRect navBarFrame = self.navigationController.navigationBar.frame;
-    UIImage *rightButtonImage = self.navigationItem.rightBarButtonItem.image;
-    CGRect navBarLabelFrame = CGRectMake(0,
-                                         0,
-                                         navBarFrame.size.width - rightButtonImage.size.width,
-                                         navBarFrame.size.height);
-    self.navBarLabel.frame = navBarLabelFrame;
+    if (@available(iOS 26, *)) {
+        // No-op - skipping code in else block for iOS 26 because the view controller doesn't display with it
```

Comment:
This whole issue is odd to me, I found this work around pretty much through brute force by checking what was different between this view controller and another working one. Since it’s just the sample app I was leaning toward putting it in to unblock testing with this app but can also spend more time on it if anyone has ideas

Tested on iOS 18 & 26
<p float="left">
<img width="300" height="1064" alt="Screenshot 2025-09-17 at 1 03 52 PM" src="https://github.com/user-attachments/assets/baeae349-7d19-4987-bbd2-98d0caa48930" />
<img width="300" height="1062" alt="Screenshot 2025-09-17 at 5 25 19 PM" src="https://github.com/user-attachments/assets/140f1447-2406-452a-81b4-b9b9e3365c14" />
</p>